apps/gitness/src/routes.tsx

```diff
@@ -102,6 +104,26 @@ export const routes: CustomRouteObject[] = [
                       routeName: RouteConstants.toRepoCommits
                     }
                   },
+                  {
+                    path: 'commits/:commitSHA',
```

Comment:
can't we make this a child of the `/commit` path? that'll simplify the breadcrumbs significantly